src/ContainerAppHelper.ts

```diff
@@ -5,8 +5,8 @@ import { GitHubActionsToolHelper } from './GitHubActionsToolHelper'
 import fs = require('fs');
 
 const ORYX_CLI_IMAGE: string = 'mcr.microsoft.com/oryx/cli:builder-debian-bullseye-20230926.1';
-const ORYX_BULLSEYE_BUILDER_IMAGE: string = 'mcr.microsoft.com/oryx/builder:debian-bullseye-20231107.2'
-const ORYX_BOOKWORM_BUILDER_IMAGE: string = 'mcr.microsoft.com/oryx/builder:debian-bookworm-20231107.2'
+const ORYX_BULLSEYE_BUILDER_IMAGE: string = 'mcr.microsoft.com/oryx/builder:aca-debian-bullseye-20240124.1'
```

Comment:
Thank you for updating these!
Could you please remove the `aca-` from the Bullseye builder? This is for the ACA Builder integrated with the Cloud Build, whereas the ones used here are just the base builders.